plugin/trino-opensearch/src/main/java/io/trino/plugin/opensearch/decoders/BigintDecoder.java

```diff
@@ -38,7 +41,7 @@ public BigintDecoder(String path)
     }
 
     @Override
-    public void decode(SearchHit hit, Supplier<Object> getter, BlockBuilder output)
+    public void decode(SearchHit hit, Supplier<Object> getter, BlockBuilder output, List<String> dereferenceName)
```

Comment:
Rename to `dereferenceNames` or `dereferencePath` as suggested by Marius.

Reply:
done